lib/l10n/intl_en.arb

```
@@ -351,5 +351,10 @@
           "type": "int"
         }
       }
-    }
+    },
+    "Find_Group": "Find Group",
+    "Discover_Groups": "Discover Groups",
+    "Search_for_public_groups": "Search for public groups",
+    "Members": "Members",
```

RuboCop (1.73)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)